middle_end/flambda/from_lambda/cps_conversion.ml

```diff
@@ -1044,10 +1049,15 @@ and cps_switch (switch : L.lambda_switch) ~scrutinee (k : Continuation.t)
           failaction = None;
         }
       in
-      let isint_switch = cps_non_tail
-        (L.Lprim (Pflambda_isint, [L.Lvar scrutinee], Loc_unknown))
-        (fun scrutinee -> I.Switch (scrutinee, isint_switch))
-        k_exn
+      let is_scrutinee_int = Ident.create_local "is_scrutinee_int" in
+      let isint_switch = I.Let ( is_scrutinee_int,
```

Comment:
Same as above, and while you're here you should probably fix the indentation for the code below.